### ISSUE TITLE
Sub menu offset wrong

### ISSUE THREAD

**Issue by alibarut** [jonataswalker/ol-contextmenu#215]
Submenu offset is different every time i right click. I found solution please update repo.

The **offset** function was not giving correct results because the element top css value changed each time. I reset the top offset again before the getViewportSize function is called and the problem is solved.

I add only at the row 424 `sub.style.top = "0px";`  
```
          subs.forEach(function (sub) {
                sub.style.top = "0px";
                // is there enough space for submenu height?
                var viewport = getViewportSize();
                var sub_offset = offset(sub);
                var sub_height = sub_offset.height;
                var sub_top = space_left_h - sub_height;

                if (sub_top < 0) {
                    sub_top = sub_height - (viewport.h - sub_offset.top);
                    sub.style.top = "-" + sub_top + "px";
                }
                sub.style.left = this$1.submenu.lastLeft;
            });
```

**Comment by jonataswalker** [comment]
Hi @alibarut 

I've just published `5.0.0-alpha`. Are you able to help with a validation?

**Comment by jonataswalker** [comment]
Should be solved with #219
